plugins/roll.py

```diff
@@ -25,9 +25,11 @@ def roll_number(message, number):
                 message.reply("{} :chuckles:".format(random))
         else:
             message.reply('Spa leur smh')
-    elif number > 1:
+    elif number > 2:
```

Comment:
```suggestion
    elif number > 1:
```
S'pas nécessaire. Tu check déjà number == 1